annotation-processing/src/main/resources/creator.mustache

```diff
@@ -0,0 +1,10 @@
+package {{packageName}};
```

Comment:
mustache was initially made for html and thus the output is HTML escaped by default. We don't need it here. Triple mustaches `{{{packageName}}}` instruct mustache to output unescaped value.